### PR TITLE
Update device delete confirmation message

### LINKED ISSUE #4100: Change the pop-up message when deleting the device from the console.
#### Summary
While deleting the end-device from the console, the pop-up message in the console displays that we cannot reuse the `device ID`.

#### Steps to Reproduce
1. Create a device in the console
2. Delete the device via console

#### What do you see now?
While deleting the device via console, you will observe the pop-up message as `Are you sure you want to delete "xxx"? This action cannot be undone and it will not be possible to reuse the end device ID`.
device-id_deletion

#### What do you want to see instead?
In TTS V3, when an end device is deleted, the ID and EUI are released, and we can reuse them. Remove the notification as it is no more needed.
Ref: https://www.thethingsindustries.com/docs/reference/id-eui-constraints/

#### Environment
TTS v3.12.1

#### How do you propose to implement this?

...

#### How do you propose to test this?

...

#### Can you do this yourself and submit a Pull Request?
No. @kschiffer

### PR DESCRIPTION
**Pull request by asmulko**
#### Summary

Closes #4100 

#### Changes

- Updated device delete confirmation message

Before:
device_delete_before

After:
device_delete_after

#### Testing

- Manually tested

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/views/device-general-settings/identity-server-form/index.js [hunk 1]
```diff
@@ -38,8 +38,7 @@ import validationSchema from './validation-schema'
 
 const messages = defineMessages({
   deleteDevice: 'Delete end device',
-  deleteWarning:
-    'Are you sure you want to delete "{deviceId}"? This action cannot be undone and it will not be possible to reuse the end device ID.',
+  deleteWarning: 'Are you sure you want to delete "{deviceId}"?',
```
Comment (kschiffer) [inline pkg/webui/console/views/device-general-settings/identity-server-form/index.js]:
```suggestion
  deleteWarning: 'Are you sure you want to delete "{deviceId}"? This action cannot be undone.',
```

I think it's still good to make users aware that the IDs are typically not reusable right away but after a retention time only.

cc @htdvisser

**Review by kschiffer — CHANGES_REQUESTED** [review]
Please also adjust the messages for deleting all other entities (gateways, organizations, applications, users).

File: pkg/webui/console/views/device-general-settings/identity-server-form/index.js (hunk 1 above)
Comment (htdvisser) [reply pkg/webui/console/views/device-general-settings/identity-server-form/index.js]: See https://www.thethingsindustries.com/docs/reference/id-eui-constraints/#deleted-entities

> When an end device is deleted, the ID and EUI are released and can be reused

Comment (kschiffer) [reply pkg/webui/console/views/device-general-settings/identity-server-form/index.js]: I see! See my updated change suggestion then @asmulko

**Review by kschiffer — CHANGES_REQUESTED** [review]
~~This is still adjusting the device modal only.~~

> Please also adjust the messages for deleting all other entities (gateways, organizations, applications, users).

**Comment by kschiffer** [comment]
Disregard the above. Misunderstanding, I'll approve.

**Review by kschiffer — APPROVED** [review]

**Pull request merged**